src/store/modules/job/mutations.ts

```diff
@@ -30,7 +30,7 @@ const mutations: MutationTree <JobState> = {
     [types.JOB_DESCRIPTION_UPDATED] (state, enums) {
         if (enums) {
             enums.forEach((enumId: any) => {
-              state.enumIds[enumId.enumId] = enumId.description
+              state.enumIds[enumId.enumId] = enumId
```

Comment:
```suggestion
              state.enumIds[enumId.enumId] = enum
```

Reply:
Renamed it enumInfo as enum is a keyword.